setup.md

```diff
@@ -121,108 +121,35 @@ $ git config --global core.editor "nano -w"
 ~~~
 {: .language-bash}
 
-### GitHub Account
-GitHub is a free, online host for Git repositories that you will use during the course to store your code in so 
-you will need to open a free [GitHub](https://github.com/) account unless you do not already have one.
+### GitLab Account
+GitLab is an online host for Git repositories that you will use during the course to store your code in.
```

Comment:
Just to double-check: did you remove the `free` intentionally?
The word might cause confusion due to its ambiguity anyway, so I think it's good to remove it.

Reply:
Yes, intentionally! It can be free, but they are using an enterprise instance, which is not free I think.